packages/react/src/stories/PlanDefinitionBuilder.stories.tsx

```diff
@@ -21,3 +22,14 @@ export const Basic = (): JSX.Element => (
     />
   </Document>
 );
+
+export const Covid19Eval = (): JSX.Element => {
+  return (
+    <PlanDefinitionBuilder
+      value={Covid19CarePlanDefinition}
```

Comment:
Looking at the storybook, looks like the PD.actions do not have timings.  Possible to add them?